src/server/game/Entities/Player/Player.cpp

```diff
@@ -7006,7 +7006,11 @@ void Player::ApplyItemEquipSpell(Item* item, bool apply, bool form_change)
             continue;
 
         // Spells that should stay on the caster after removing the item.
-        constexpr std::array<int32, 1> spellExceptions = { /*Electromagnetic Gigaflux Reactivator*/ 11826 };
+        constexpr std::array<int32, 2> spellExceptions =
+        {
+            /*Electromagnetic Gigaflux Reactivator*/ 11826,
+            /*Book of the Dead - Summon Skeleton*/ 17490
```

Comment:
trivial, but the comment would be preferred to be after the spellID

Reply:
Noticed that, wasn't sure whether to go with existing formatting or what. Fixed now.